pages/common/jpegoptim.md

```diff
@@ -13,3 +13,7 @@
 - Force the output images to be progressive:
 
 `jpegoptim --all-progressive {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Force the output images to be fixed max size:
```

Comment:
I think "fixed upto a max size" is grammatically correct.

Reply:
Perhaps `to have a fixed maximum size` would read better?